src/i18n/en-US/modelPlan/modelToOperations.ts

```diff
@@ -42,6 +42,27 @@ export const modelToOperationsMisc: Record<string, any> = {
       description: '18 categories, 0 milestones, 0 solutions',
       buttonText: 'Use this template'
     }
+  },
+  modal: {
+    title: 'Add a new {{type}}',
+    allFieldsRequired:
+      'Fields marked with an asterisk ( <s>*</s> ) are required.',
+    category: {
+      selectPrimaryCategory: {
+        label: 'Select primary category <s>*</s>',
+        sublabel:
+          'Choose a primary category if you are adding a sub-category, or choose "None" if you are adding a primary category.'
+      },
+      categoryTitle: {
+        label: 'New category title <s>*</s>'
+      },
+      sortOptions: {
+        default: '- Select - ',
+        none: 'None (this is a primary category)'
+      }
+    },
+    addButton: 'Add {{type}}',
+    cancel: 'Cancel'
```

Comment:
For these form questions, we want to handle them to be used in change history.  This includes adding them to `types/translation.ts`.   I don't think we have anything set up yet, but the tables will need to be created and also added to the export util.  I can pair on this if you need!